lib/wiki_course_edits.rb

```diff
@@ -101,7 +85,6 @@ def remove_assignment(assignment:)
 
   private
 
-  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Will this work still by removing the comment for rubocop at this line and one below to disable/enable Metrics/CyclomaticComplexity?
Running rubocop locally with the comments in place seems to produce offenses.